osc-server/src/main/java/org/osc/core/broker/service/AddDeploymentSpecService.java

```diff
@@ -93,6 +93,7 @@ public BaseJobResponse exec(BaseRequest<DeploymentSpecDto> request, EntityManage
                 BaseJobResponse response = new BaseJobResponse();
                 response.setId(ds.getId());
                 LockUtil.releaseLocks(unlockTask);
+                return response;
```

Comment:
on line 86 to be consistent return new BaseJobResponse(this.ds.getId(), job.getId())